custom_components/tuya_local/devices/benexmart_blind_motor.yaml

```diff
@@ -0,0 +1,26 @@
+name: benexmart blind motor
+primary_entity:
+  entity: cover
+  class: blind
+  dps:
+    - id: 1
+      name: control
+      type: string
+      mapping:
+        - dps_val: open
+          value: open
+        - dps_val: close
+          value: close
+        - dps_val: stop
+          value: stop
+    - id: 2
+      name: position
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - invert: true
+    - id: 5
```

Comment:
I think if you call this "reversed", you can remove the invert: true mapping from position.  You may need to add a mapping to swap true and false if the "reverse" direction is the opposite of what is expected.